modules/network/src/main/java/org/apache/ignite/internal/network/netty/PipelineUtils.java

```diff
@@ -57,6 +58,9 @@ public static void setup(ChannelPipeline pipeline, PerSessionSerializationServic
      */
     public static void setup(ChannelPipeline pipeline, PerSessionSerializationService serializationService,
             HandshakeManager handshakeManager, Consumer<InNetworkObject> messageListener) {
+        // Consolidate flushes to bigger ones (improves throughput with smaller messages at the price of the latency).
```

Comment:
And if there are big messages and small ones, will they also be collected in batch or will there be a flush immediately if there are big ones?

Reply:
Big messages are split to fragments up to 16kb each, and those fragments will be collected to batches along with small messages. Batch size is 256 at max, so this will take up to 4Mb per event loop (and number of event loops is cpus*2).